ambry-store/src/test/java/com.github.ambry.store/StoreTestUtils.java

```diff
@@ -36,6 +46,87 @@ static File createTempDirectory(String prefix) throws IOException {
   }
 
   /**
+   * Need mock ReplicaId to get and set isSealed state
+   */
+  private static class MockReplicaId implements ReplicaId {
```

Comment:
can't the existing impl be upgraded to do this?

Reply:
Previous implementation used Mockito to create the mock; Mockito is cumbersome and problematic when trying to add stateful methods.  Other MockReplicaId in clustermap package doesn't let me define the methods I need defined.